### ISSUE TITLE
Compilation issue (version mismatch with V-REP headers)

### ISSUE THREAD

**Issue by fferri** [CoppeliaRobotics/v_repExtRosInterface#2]
_From @HerveFrezza-Buet on July 18, 2016 14:53_

Thus is a compilation issue report :

vrep_ros_interface.cpp:647:48: error: ‘sim_boolparam_rosinterface_donotrunmainscript’ was not declared in this scope

It seems to be solved in the package provided by V-REP.

Thanks for your contribution, I hope my minor one will help.

_Copied from original issue: fferri/v_repExtRosInterface#2_

**Comment by fferri** [comment]
sim_boolparam_rosinterface_donotrunmainscript  is defined in programming/include/v_repConst.h.

Either you have an old V-REP version, or you didn't copy v_repConst.h/v_repTypes.h/v_repLib.h/cpp from V-REP to the package dir.

Ideally these files should not be copied to the ROS pkg dir, but rather referenced. So I'll leave the issue open until this has been changed.

**Comment by fferri** [comment]
_From @HerveFrezza-Buet on July 19, 2016 8:4_

Indeed, it worked from the V-REP directory. I was attempting to get the Ros Interface from this git repo. And here, in your include/v_repConst.h, it is actually missing.

**Comment by fferri** [comment]
Since V-REP itself it's not a ROS package, the only way to reference files from it is by setting a variable which tells where is V-REP installed.

I opted for an environment variable ($VREP_ROOT), so you can set it once in your ~/.bashrc and requires no changing of CMakefile.txt's.

Could you test it again?

**Comment by fferri** [comment]
_From @HerveFrezza-Buet on July 19, 2016 12:53_

I did, and it worked. But I had to change the comments in the CMakelist at V-REP side for building the library. 

Then, I had to copy ros-ws/devel/lib/libv_repExtRosInterface.so into VREP_ROOT.

This is straightforward... once you know ho to proceed. Maybe a re-writing of the readme (and the vrep doc) for this install should help significantly. 

Many thanks for your work.
